core/src/test/java/org/apache/calcite/avatica/util/TimestampFromNumberAccessorTest.java

```diff
@@ -101,6 +101,9 @@ public class TimestampFromNumberAccessorTest {
     value = DateTimeUtils.timestampStringToUnixDate("2014-09-30 15:28:27.356");
     assertThat(instance.getString(), is("2014-09-30 15:28:27"));
 
+    value = DateTimeUtils.timestampStringToUnixDate("2014-2-2 15:28:27.356");
```

Comment:
Could you add test case to cover `'1945-2-2 12:2:5'` and `'12:2:5' to test the TIME type without leading zero?

Reply:
@herunkang2018 done.